src/styles/atoms/_atoms.button.scss

```diff
@@ -413,3 +413,39 @@
     width: 100%;
   }
 }
+
+/**
+ * BUTTON WITH AVATAR
+ * -------------------------------------------------------------------
+ */
+
+.a-button {
+  &.has-avatar,
+  &.has-avatar-with-image {
+    position: relative;
+
+    .a-avatar {
+      left: 0;
+      position: absolute;
+      top: 0;
+    }
+  }
+
+  &.has-avatar {
+    padding-left: var(--spacer-xl);
```

Comment:
I.p.v. spacers is het misschien beter om hier een berekening te maken met de afmetingen van de avatar, anders kunnen hier discrepanties ontstaan. Opmerking geldt ook voor verdere code.